sections/faqs/html-attribute-warnings.js

```diff
@@ -0,0 +1,65 @@
+import md from 'components/md'
+
+const HTMLAttributeWarnings = () => md`
+  ## Why am I getting HTML attribute warnings?
+
+  The warning message below indicates that non-standard attributes are being attached to
+  HTML DOM elements such as \`div\` or \`a\`. If you are seeing this warning message, it is likely that
+  you or a library you are using is attaching props as attributes to HTML DOM elements.
+
+  \`\`\`sh
+  Warning: Received 'true' for a non-boolean attribute
+  \`\`\`
+
+  ### Styling a Standard React Component with \`styled()\`
```

Comment:
I'm not sure what you're getting at in this section; It seems that it's kind of disconnected from the above. You could maybe just write.

> If you're seeing this warning you are probably passing `true` where `"true"` would be appropriate.
> It's likely that this comes from a `.attrs({})` property, or from a completely unrelated prop that you're passing to a `styled(Component)` component.
> In the latter case we're passing through all props, so if it so happens that you have named a prop something that is valid html and didn't mean to pass it through, make sure you're not passing it on to a DOM element.
> And if you're meaning to pass this prop to the DOM element, make sure to follow the warning and adapt the value. The  example message above would indicate that instead of `true` you should pass on `"true"`, a string.

Reply:
I've made the changes, but apparently, Github Desktop's amazing commit interface has changed (had not used it in a while, haha). I will push up a force commit with all the changes which should have a cleaner tree.